docs/_posts/2021-12-10-seminar-minutes-2021-11.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Bot Friday 2021年第11次活动纪要
+author: neilzwh
```

Comment:
In order to show your profile in the blog, please add your profile in this file: https://github.com/wechaty/bot5/blob/main/docs/_data/authors.yml

Reply:
Already update author information. Thanks.